packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -828,6 +832,200 @@ describe("Integration tests", () => {
+          updatedAt: new Date(Number(writtenPayload.tenant?.updatedAt)),
+        };
+        expect(writtenPayload.tenant).toEqual(toTenantV2(updatedTenant));
+      });
+      it("Should throw tenant not found", async () => {
+        await addOneAttribute(attribute, attributes);
+        expect(
+          tenantService.revokeCertifiedAttributeById(
+            targetTenant.id,
+            attributeId,
+            mockAuthData,
+            correlationId
+          )
+        ).rejects.toThrowError(tenantNotFound(requesterTenant.id));
+      });
+      it("Should throw attribute not found", async () => {
+        await addOneTenant(targetTenant, postgresDB, tenants);
+        await addOneTenant(requesterTenant, postgresDB, tenants);
+
+        expect(
+          tenantService.revokeCertifiedAttributeById(
+            targetTenant.id,
+            attributeId,
+            mockAuthData,
+            correlationId
+          )
+        ).rejects.toThrowError(attributeNotFound(attribute.id));
+      });
+      it("Should throw tenant is not a certifier", async () => {
+        const requesterTenant: Tenant = {
+          ...mockTenant,
+          updatedAt: currentDate,
+          name: "A requesterTenant",
+        };
+        await addOneAttribute(attribute, attributes);
+        await addOneTenant(targetTenant, postgresDB, tenants);
+        await addOneTenant(requesterTenant, postgresDB, tenants);
+
+        expect(
+          tenantService.revokeCertifiedAttributeById(
+            targetTenant.id,
+            attributeId,
+            mockAuthData,
+            correlationId
+          )
+        ).rejects.toThrowError(tenantIsNotACertifier(requesterTenant.id));
+      });
+      it("Should throw certifiedAttributeOriginIsNotCompliantWithCertifier", async () => {
+        const notCompliantOriginAttribute: Attribute = {
+          ...attribute,
+          origin: generateId(),
+        };
+        await addOneAttribute(notCompliantOriginAttribute, attributes);
+        await addOneTenant(targetTenant, postgresDB, tenants);
+        await addOneTenant(requesterTenant, postgresDB, tenants);
+
+        expect(
+          tenantService.revokeCertifiedAttributeById(
+            targetTenant.id,
+            attributeId,
+            mockAuthData,
+            correlationId
+          )
+        ).rejects.toThrowError(
+          certifiedAttributeOriginIsNotCompliantWithCertifier(
+            notCompliantOriginAttribute.origin!,
+            requesterTenant.id,
+            targetTenant.id,
+            requesterTenant.features[0].certifierId
+          )
+        );
+      });
+      it("Should throw certifiedAttributeAlreadyRevoked", async () => {
+        const tenantAlreadyRevoked: Tenant = {
+          ...targetTenant,
+          attributes: [
+            {
+              id: attribute.id,
+              type: "PersistentCertifiedAttribute",
+              assignmentTimestamp: new Date(),
```

Comment:
```suggestion
              ...attribute
```
You can get the default values from the object defined before and here you can set the `revocationTimestamp`, which is the only field useful to this test case

Reply:
This is a `TenantAttribute`, not an `attribute`, so I used `mockCertifiedTenantAttribute`, leaving the id as `attribute.id`